conf/submarine-site.xml.template

```diff
@@ -155,4 +155,28 @@
     <description>Rpc server port</description>
   </property>
 
+  <!-- Submarine Submitters Configuration  -->
+    <property>
+      <name>submarine.submitters</name>
+      <value>k8s</value>
+      <description>Submitter list for the server</description>
+    </property>
+    <property>
+      <name>submarine.submitters.k8s.class</name>
+      <value>org.apache.submarine.server.submitter.k8s.K8sJobSubmitter</value>
+      <description>The entry class for the specified submitter</description>
+    </property>
+    <property>
+      <name>submarine.submitters.k8s.classpath</name>
+      <value>../lib/submitter/k8s/</value>
+      <description>The libs for Kubernetes submitter</description>
+    </property>
+
+    <!-- K8s Configuration -->
+    <property>
+      <name>submarine.k8s.kube.config</name>
```

Comment:
k8s cluster already have `KUBECONFIG`, When testing locally, you can get and set environment variables through  `KUBECONFIG`.

Reply:
Sorry for that, I forgot to remove it. In cluster mode, we don't need this.